docs/pjrt.md

```diff
@@ -194,15 +194,61 @@ for more information.
 
 *Warning: GPU support is still highly experimental!*
 
+### Single-node GPU training
+
 To use GPUs with PJRT, simply set `PJRT_DEVICE=GPU` and configure
 `GPU_NUM_DEVICES` to the number of devices on the host. For example:
 
 ```
 PJRT_DEVICE=GPU GPU_NUM_DEVICES=4 python3 xla/test/test_train_mp_imagenet.py --fake_data --batch_size=128 --num_epochs=1
 ```
 
-Currently, only a single host is supported, and multi-host GPU cluster support
-will be added in an future release.
+You can also use `torchrun` to initiate the single-node multi-GPU training. For example,
+
+```
+PJRT_DEVICE=GPU torchrun --nnodes 1 --nproc-per-node ${NUM_GPU_DEVICES} xla/test/test_train_mp_imagenet.py --fake_data --batch_size=128 --num_epochs=1
+```
+
+In the above example, `nnodes` means how many machines (physical machines or VMs) to be used (it is 1 since we do single-node training). `NUM_GPU_DEVICES` means how many GPU devices to be used.
+
+### Multi-node GPU training
+
+Note that this feature only works for cuda 12+. Similar to how PyTorch uses multi-node training, you can run the command as below:
+
+```
+PJRT_DEVICE=GPU torchrun \
+--nnodes=${NUMBER_GPU_VM} \
+--node_rank=${CURRENT_HOST_RANK} \
+--nproc_per_node=${NUMBER_LOCAL_GPU_DEVICES} \
+--rdzv_endpoint=<internal_ip_address:port> multinode_training.py
+```
+
+`--nnodes`: how many GPU machines to be used.
```

Comment:
This paragraph is getting mangled: 
![image](https://github.com/pytorch/xla/assets/15197278/0da39f2a-c61e-4572-9c6a-165fa8c97666)

Use an unordered list, [table, or definition list](https://gist.github.com/stevenyap/7038119#table-and-definition-list)

Reply:
Thanks for the catch!